docs/pages/components/side-navigation.md

```diff
@@ -457,54 +465,55 @@ The following requirements must be met:
 
 {% capture default %}
 <nav class="fd-side-nav fd-side-nav--condensed">
+    <a class="fd-side-nav__skip-link" href="#content">Skip navigation</a>
     <div class="fd-side-nav__main-navigation">
         <ul class="fd-nested-list fd-nested-list--compact">
             <li class="fd-nested-list__item">
                 <a class="fd-nested-list__link" href="#">
-                    <span class="fd-nested-list__icon sap-icon--home"></span>
+                    <span aria-label="Level 1 Item" class="fd-nested-list__icon sap-icon--home"></span>
                     <span class="fd-nested-list__title">Level 1 Item</span>
                 </a>
             </li>
             <li class="fd-nested-list__item">
                 <a class="fd-nested-list__link" href="#">
-                    <span class="fd-nested-list__icon sap-icon--calendar"></span>
-                    <span class="fd-nested-list__title">Level 1 Item</span>
+                    <span aria-label="Level 1 Item" class="fd-nested-list__icon sap-icon--calendar"></span>
+                    <a class="fd-nested-list__title" href="#">Level 1 Item</a>
```

Comment:
Is this supposed to be an anchor in an anchor, as it's not a child item?

Reply:
Fixed the markup